static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -76,15 +76,20 @@ const Context = ({
 
   // Temporarily allow mobile platforms to make API call and "show" stacktrace link
   if (isMobile) {
-    return (
-      <ErrorBoundary customComponent={null}>
-        <StacktraceLink
-          line={frame.function ? frame.function : ''}
-          frame={frame}
-          event={event}
-        />
-      </ErrorBoundary>
-    );
+    if (
+      event.platform !== 'java' ||
+      (event.platform === 'java' && frame?.module?.substring(0, 4) === 'com.')
+    ) {
```

Comment:
Check that the frame module begins with `com.` for mobile java projects.

This if statement was not combined with the previous one so it can be removed more easily later